laravel5/app/Monomalpoly/CaseDeJeux.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class CaseDeJeux
+{
+	private $type;
+	private $nom;
+	private $listeJoueurs;
```

Comment:
Should the game box really have a list of players? I thought it was the player's pawn which was on a box, not the other way around.